packages/fxa-settings/.rescriptsrc.js

```diff
@@ -0,0 +1,59 @@
+const { resolve } = require("path");
+
+const additionalJSImportPaths = [resolve("../fxa-components")];
+
+const permitAdditionalJSImports = config => {
+  // Update ModuleScopePlugin's appSrcs to allow our new directory
+  config.resolve.plugins.forEach(plugin => {
+    if (plugin.constructor && plugin.constructor.name === "ModuleScopePlugin") {
+      plugin.appSrcs.push(...additionalJSImportPaths);
+    }
+  });
+
+  // We need to target a loader that handles compiling the JS/TS/JSX/TSX files, which exists
+  // as a nameless object at a very arbitrary location. Without this we would still be able
+  // to import the file but it wouldn't get transpiled. However, CRA is a very stable project
+  // so I don't expect this object's location or position in the array of loaders to change
+  // too frequently. Here's what it looks like:
+  //
+  // {
+  //   module: {
+  //     strictExportPresence: true,
+  //     rules: [
+  //       { ... }, // unrelated item
+  //       { ... }, // unrelated item
+  //       {
+  //         oneOf: [
+  //           { ... }, // unrelated item
+  //           {
+  //             test: /\.(js|mjs|jsx|ts|tsx)$/,
+  //             include: '/base/path/fxa/packages/fxa-settings/src', // <- This is the property we want to modify
+  //             loader: '/base/path/fxa/packages/fxa-settings/node_modules/babel-loader/lib/index.js',
+  //             options: [Object]
+  //           },
+  //         ]
+  //       }
+  //     ]
+  //   }
+  // }
+  if (
+    config.module.rules[2] &&
+    config.module.rules[2].hasOwnProperty("oneOf") &&
+    config.module.rules[2].oneOf[1].test.toString() ===
```

Comment:
This is fine. If path churn here becomes a problem maybe [jsonpath](https://www.npmjs.com/package/jsonpath) could help.